app/views/password_resets/show.html.haml

```diff
@@ -0,0 +1,10 @@
+%section.reset_password.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_tag url: password_resets_path do |f|
+        %header
+          %h1 Reset Your Password
+        %fieldset.form-group
+          = f.text_field :password, label: "New Password"
```

Comment:
Well.. passwords should be covered, you should always use `password_field` for this matter.